ts/Stock/RangeSelector/RangeSelector.ts

```diff
@@ -1450,11 +1473,25 @@ class RangeSelector {
         delete buttonTheme.width;
         delete buttonTheme.states;
 
+        const langOptions = this.chart.options.lang;
 
         this.buttonOptions.forEach((
             rangeOptions: RangeSelectorButtonOptions,
             i: number
         ): void => {
+            if (rangeOptions.type && langOptions.rangeSelector) {
+                rangeOptions.text ??= langOptions.rangeSelector[`${rangeOptions.type}Text`];
+                rangeOptions.title ??= langOptions.rangeSelector[`${rangeOptions.type}Title`];
+            }
```

Comment:
Does this part seem redundant? Or is it a fix for the missing title translation I reported some time ago? I mean, in lines 473-775 (`init()` method) we have exactly the same logic.

Reply:
Seems so 👍